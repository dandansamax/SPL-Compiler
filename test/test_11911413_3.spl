
struct a{
  int 0str_a;
} str;

int test_11911413_3(){

    char ccc=0x23;
    
   return 0;
}