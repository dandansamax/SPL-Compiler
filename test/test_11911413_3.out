Error type A at Line 3: unknown lexeme 0str_a
